### ISSUE TITLE
[Feature Request]: Randomization

### ISSUE THREAD

**Issue by jubera** [jasp-stats/jasp-issues#1815]
### Description

Implementation of randomization techniques

### Use-case

Clinical trial studies (among others)

### Is your feature request related to a problem?

It gets complicated when you have several factors (e.g., sex, age, etc) with different levels to assign blindly each participant to a "treatment" group

### Additional context

Random allocation is a method used to assign participants to different research groups 

There are different ways to it: stratified, pair-matched,...

There are also packages in R to do it (but I do not have experience with them)

**Comment by juliuspf** [comment]
Hi @jubera, thank you for the request. @EJWagenmakers this seems like more of a study planning tool instead of data analysis. Does this make sense for JASP?

**Comment by EJWagenmakers** [comment]
It does. In fact, our Quality Control module has "DOE", design of experiments, and this is conceptually close.

**Comment by tomtomme** [comment]
@JTPetter 
Is this feature covered nowadays via DOE or is this planned?

**Comment by JTPetter** [comment]
@tomtomme No, it is neither covered nor planned at the moment. I don't think it would fit in DoE. As I understand EJ I believe he only meant that this would fit inside of the scope of JASP as DOE is also used for planning. 

Not 100% sure where this would fit, maybe even a dedicated 'randomization' module that can add an "assigned group" column to an existing dataset based on different parameters (e.g., number of groups, proportion per group) and possibly stratify based on other variables.
